lang/fr/texts.php

```diff
@@ -5300,6 +5300,8 @@
     'local_domain_help' => 'EHLO domain (optional)',
     'port_help' => 'ie. 25,587,465',
     'host_help' => 'ie. smtp.gmail.com',
+    'btcpay_refund_subject' => 'Remboursement de votre facture via BTCPay',
+    'btcpay_refund_body' => 'Un remboursement qui vous est destiné a été émis. Pour l\'obtenir via BTCPay, veuillez cliquer sur ce lien:',
 );
```

Comment:
Hi There,

We will lose this translation if placed here. If you can insert an english translation in lang/en/texts.php

this will be the best place, we can then support additional language translations of this key.